File: online-judge-solutions/Codeforces/1722A.py
#https://codeforces.com/problemset/problem/1722/A
#Lang		:	Python 3.8
#Time		:	46 ms
#Memory         :	0 KB
for _ in range(int(input())):
    n,s,nombre=int(input()),input(),"Timru"
    s=''.join(sorted(s))
    print("YES" if s==nombre else "NO")

File: online-judge-solutions/Codeforces/1722A.cpp
//https://codeforces.com/problemset/problem/1722/A
//Lang		:	GNU C++14
//Time		:	15 ms
//Memory	:	0 KB
#include<iostream>
#include<algorithm>
using namespace std;
int main(){
	int t,n;
	cin>>t;
	while(t--){
		string s,nombre="Timru";
		cin>>n>>s;
		sort(begin(s),end(s));
		cout<<(s==nombre?"YES":"NO")<<"\n";
	}
	return 0;
}
